### PR TITLE
fix issue #2635

### PR DESCRIPTION
**Pull request by saransh-dev**
# Fixed issue #2635 

### Changes
apinf_packages/branding/cover_photo/client/view/view.html
apinf_packages/branding/logo/client/view/view.html

Closes: #2635

### CODE REVIEW

File: apinf_packages/branding/cover_photo/client/view/view.html [hunk 1]
```diff
@@ -5,6 +5,10 @@
 
 <template name="viewCoverPhoto">
   {{# if coverPhotoExists }}
-  <img class="view-cover-photo" src="{{ uploadedCoverPhotoLink }}" />
+  <div class="form-group">
+	  <label for="branding-site-cover-photo" class="control-label">Cover Image</label>
```
Comment (marla-singer) [inline apinf_packages/branding/cover_photo/client/view/view.html]: 1. Use i18n tag for text
2. Name for tag have to be as `{{_ 'templateName_element_meanValue' }}`
For example, `{{_ 'viewCoverPhoto_labelText_coverImage' }}`
3. Add value for tag only for English text, related file is `en.i18n.json`
2. Place it as nested element
```html
<label for="branding-site-cover-photo" class="control-label">
  {{_ 'viewCoverPhoto_labelText_coverImage' }}
</label>
```

File: apinf_packages/branding/logo/client/view/view.html [hunk 2]
```diff
@@ -5,6 +5,10 @@
 
 <template name="viewProjectLogo">
   {{# if projectLogoExists }}
-  <img class="view-project-logo" src="{{ uploadedProjectLogoLink }}"/>
+  <div class="form-group">
+	  <label for="branding-site-logo" class="control-label">Site Logo</label>
```
Comment (marla-singer) [inline apinf_packages/branding/logo/client/view/view.html]: The same stuff for this text

**Comment by matleppa** [comment]
Oh, I see that @marla-singer has add comments while my review was open. Well, her comments are more prompt than mine ;-)

File: apinf_packages/core/lib/i18n/en.i18n.json [hunk 3]
```diff
@@ -475,6 +475,8 @@
   "profile_updatedSuccess": "Profile updated",
   "profile_usernameInvalid": "Username must be 3-15 alphanumeric characters. Underscore and hyphens are allowed.",
   "profile_usernameIsTaken": "User name already exists. Enter another user name.",
+  "projectLogoTitle_logoTitle":"Site Logo",
+  "projectCoverPhotoTitle_coverPhotoTitle":"Cover Image",
```
Comment (marla-singer) [inline apinf_packages/core/lib/i18n/en.i18n.json]: 1. Re-name tags names as "branding_projectLogoTitle_logoTitle" and "branding_projectCoverPhotoTitle_coverPhotoTitle"
2. Add a spacebar after `:`

File: apinf_packages/branding/client/branding.html [hunk 4]
```diff
@@ -17,10 +17,22 @@ <h3 class="panel-title">
           </div>
           <div class="panel-body">
             {{# if branding }}
-              {{> viewProjectLogo branding=branding }}
-              {{> uploadProjectLogo branding=branding }}
-              {{> viewCoverPhoto branding=branding }}
-              {{> uploadCoverPhoto branding=branding }}
+              <div class="form-group">
+                <label for="branding-site-logo" class="control-label">
+                {{_ "projectLogoTitle_logoTitle" }}
```
Comment (marla-singer) [inline apinf_packages/branding/client/branding.html]: Add two spacebars for nested element

Comment (matleppa) [reply apinf_packages/branding/client/branding.html]: = Indentation with two spacebars.

**Pull request merged**